pkgs/desktops/gnome-3/core/gnome-tour/default.nix

```diff
@@ -0,0 +1,80 @@
+{ stdenv
+, rustPlatform
+, gettext
+, meson
+, ninja
+, fetchFromGitLab
+, pkg-config
+, gtk3
+, glib
+, gdk-pixbuf
+, desktop-file-utils
+, appstream-glib
+, wrapGAppsHook
+, python3
+, gnome3
+, config
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "gnome-tour-unstable";
+  version = "2020-02-15";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "GNOME";
+    repo = "gnome-tour";
+    rev = "129ec5d20a512ed4937da5088170c8ba61579c7b";
+    sha256 = "03if1y607jhy70rgb6bdicqj16c7khaa4wmiga7jrbbayf64nlgy";
+  };
+
+  cargoSha256 = "1gh2xvaq7v0zh375f98pbhjx65b07km47n2g5lplrxvznaw03c2a";
+
+  mesonFlags = [
+    "-Ddistro_name=NixOS"
+    "-Ddistro_icon_name=nix-snowflake"
+    "-Ddistro_version=20.03"
```

Comment:
I wish I could get this value from nixos

Reply:
https://gitlab.gnome.org/GNOME/gnome-tour/issues/10